src/assets/style/mixins.scss

```diff
@@ -0,0 +1,22 @@
+@mixin gradient($from, $to) {
+  /* fallback/image non-cover color */
+  background-color: $from;
+
+  /* Firefox 3.6+ */
```

Comment:
This goes for most of this file, but CRA comes with https://github.com/postcss/autoprefixer built-in, making these lines redundant